back/spec/services/xlsx_export/value_visitor_spec.rb

```diff
@@ -494,5 +480,42 @@
         expect(visitor.visit_page(field)).to be_nil
       end
     end
+
+    describe '#visit_file_upload' do
+      let(:input_type) { 'file_upload' }
+
+      context 'when there is no value' do
+        let(:value) { nil }
+
+        it 'returns nil' do
+          I18n.with_locale('nl-NL') do
+            expect(visitor.visit_file_upload(field)).to be_nil
+          end
+        end
+      end
+
+      context 'when there is a value' do
+        let(:project) { create(:continuous_native_survey_project) }
+        let(:proposed) { create(:idea_status_proposed) }
+        let(:model) do
+          Idea.create(
```

Comment:
I think it would be better to use the ideas factory?

Reply:
We cannot use the factory, because it will create an idea with attributes that are not relevant for a survey response, like a title and a description.

Reply:
I wonder if it would make sense to have a separate factory trait to create a survey response. If we go for `Idea.create`, I would prefer to use `Idea.create!` to avoid silent creation failures.

Reply:
Done. I introduced factory `native_survey_response` in `back/spec/factories/ideas.rb`.